### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by limelect** [pyscripter/pyscripter#970]
Delphi 10.2.3 while compiling
zBase, zObjInspector ,VirtualTrees.Accessibility  missing may be more.
could not find above

[dcc32 Error] dlgSynEditOptions.pas(878): E2010 Incompatible types: 'TSpTBXTabSheet' and 'TColor'
tried  eKeyShort2.color it did not help
same on eKeyShort1.

**Comment by limelect** [comment]
Also could not find components SynHighlighterWeb,
    SynWebHtmlSyn: TSynWebHtmlSyn;
    SynWebXmlSyn: TSynWebXmlSyn;
    SynWebCssSyn: TSynWebCssSyn;
    SynCppSyn: TSynCppSyn;
    SynWebEngine: TSynWebEngine;

**Comment by pyscripter** [comment]
Please follow the instructions in https://github.com/pyscripter/pyscripter/blob/master/Compilation%20Instructions.txt and you will have no problems.  
If you have already installed some of the components you can deselect them from the multi-installer.
